lib/dotcom_web/controllers/schedule/line/helpers.ex

```diff
@@ -165,7 +165,7 @@ defmodule DotcomWeb.ScheduleController.Line.Helpers do
     RoutesRepo.get_shapes(route_id, direction_id: direction_id)
   end
 
-  @spec get_route_stops(Route.id_t(), direction_id, StopsRepo.stop_by_route()) ::
+  @spec get_route_stops(Route.id_t(), direction_id, Stops.Repo.stop_by_route()) ::
```

Comment:
I feel like using the `Stops.Repo` is wrong here, it should be the prototype of the function